test_enhanced_long_confidence.py

```
@@ -0,0 +1,216 @@
+            
+            signals.append({
+                "date": current_date,
+                "signal": signal_result["signal"],
+                "confidence": signal_result["confidence"],
+                "entry_price": entry_price,
+                "exit_price": exit_price,
+                "pnl": pnl,
+                "result": result,
+                "reason": signal_result["reason"][:100] if len(signal_result["reason"]) > 100 else signal_result["reason"]
+            })
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)